### PR TITLE
Feature/add rst

### PR DESCRIPTION
**Pull request by sam-hoffman**
This PR adds a template for one-line RST, which is automatically parsed by Sphinx and much more human readable than traditional RST. This is similar to #163 , but I hope more comprehensive. 

It also fixes a minor typo in `test/docstring/get_template.spec.ts`.

### CODE REVIEW

File: src/docstring/get_template.ts [hunk 1]
```diff
@@ -8,6 +8,8 @@ export function getTemplate(docstringFormat: string): string {
             return getTemplateFile("sphinx.mustache");
         case "numpy":
             return getTemplateFile("numpy.mustache");
+	case "oneline-rst":
```
Comment (NilsJPWerner) [inline src/docstring/get_template.ts:11]: The indentation looks weird here. Can you use spaces?

File: src/test/docstring/get_template.spec.ts [hunk 2]
```diff
@@ -23,14 +23,22 @@ describe("getTemplate()", () => {
         });
     });
 
-    context("when asked for sphinx template", () => {
+    context("when asked for numpy template", () => {
```
Comment (NilsJPWerner) [inline src/test/docstring/get_template.spec.ts:26]: Nice catch

File: src/docstring/get_template.ts (hunk 1 above)
Comment (sam-hoffman) [reply src/docstring/get_template.ts]: yes! sorry about that

File: src/test/docstring/get_template.spec.ts (hunk 2 above)
Comment (sam-hoffman) [reply src/test/docstring/get_template.spec.ts:26]: thanks!

**Comment by sam-hoffman** [comment]
hey @NilsJPWerner , wanted to bring this up again to see if i could get a review, pretty please :)

File: src/docstring/get_template.ts [hunk 3]
```diff
@@ -8,6 +8,8 @@ export function getTemplate(docstringFormat: string): string {
             return getTemplateFile("sphinx.mustache");
         case "numpy":
             return getTemplateFile("numpy.mustache");
+    case "oneline-rst":
```
Comment (NilsJPWerner) [inline src/docstring/get_template.ts:11]: The indentation still looks weird

Comment (sam-hoffman) [reply src/docstring/get_template.ts]: should be fixed now, sorry about that

**Pull request merged**